llama_index/embeddings/base.py

```diff
@@ -213,15 +223,11 @@ async def aget_queued_text_embeddings(
                 event_id = self.callback_manager.on_event_start(CBEventType.EMBEDDING)
```

Comment:
since we removed the `on_event_end` in this function, lets also remove the `on_event_start`

Reply:
Hey @logan-markewich, this is generating a different event ID for each batch to mirror the behavior of the synchronous version. For the async version, the event IDs are collected in a list (callback payloads). At the end of this function, we made the `on_event_end` call for each batch to match up the embeddings with the corresponding event IDs generated here.

Reply:
Ah I missed that, that makes sense! Thanks!